api/controllers/ClientController.js

```diff
@@ -37,19 +37,27 @@ module.exports = {
    */
   async create(request) {
     const payload = request.payload || {};
-    const client = await Client.create(payload);
-    if (!client) {
+    const client = await Client.create(payload).catch((err) => {
       logger.warn(
         '[ClientController->create] Could not create client due to bad request',
-        request,
+        {
+          request,
+          stack_trace: err.message,
```

Comment:
Sharp eye, but yes this is ok. I'm happy to leave a comment in the code why, but in this case I checked the error object from mongo and decided the message was more useful than the full stack trace, since the error is a Mongo validation problem. The stack had lots of extra junk but the `err.message` was actionable in terms of providing debugging context, e.g. "client_id field is not unique. cannot create record"

Reply:
Here's an example:

```
E11000 duplicate key error collection: local.client index: id_1 dup key: { : "fake-client5" }
```